test/unit/model/cygwin-test.js

```diff
@@ -181,27 +181,25 @@ describe('Cygwin installer', function() {
           '--packages', 'openssh,rsync']);
     });
 
-    it('should catch errors thrown during the installation', function(done) {
+    it('should catch errors thrown during the installation', function() {
       let err = new Error('critical error');
       sandbox.stub(child_process, 'execFile').yields(err);
 
       try {
         installer.installAfterRequirements(fakeProgress, success, failure);
-        done();
       } catch (error) {
         expect.fail('It did not catch the error');
       }
     });
 
-    it('should copy cygwin.exe installer in target directory', function(done) {
```

Comment:
Here is done should stay as it was, because installAfterRequirements() does not return promise.